### ISSUE TITLE
Feature request: Conference auto record when moderator joins

### ISSUE THREAD

**Issue by avimar** [signalwire/freeswitch#585]
https://github.com/signalwire/freeswitch/blob/15ad4c23e259c1c2dff58f89e9124e5f36dd2e94/src/mod/applications/mod_conference/mod_conference.c#L425

Currently, you can set a conference to `auto-record`.  You can also set a `conference-flags` of `wait-mod`.
Can we please have an option, or automatically, mix those two? If you have wait-mod set and auto-record, then auto-record won't start until the mod comes.

It's not obvious to me how `CFLAG_WAIT_MOD` is used or can get called.

Thanks!

**Comment by greenbea** [comment]
I agree this is an important feature when recording a conference to be able to start the recording when the mod enters
